recaptcha_enterprise/snippets/src/main/java/recaptcha/account_defender/AccountDefenderAssessment.java

```diff
@@ -56,9 +59,16 @@ public static void main(String[] args) throws IOException, NoSuchAlgorithmExcept
     // Unique ID of the customer, such as email, customer ID, etc.
     String userIdentifier = "default" + UUID.randomUUID().toString().split("-")[0];
 
-    // Hash the unique customer ID using HMAC SHA-256.
-    MessageDigest digest = MessageDigest.getInstance("SHA-256");
-    byte[] hashBytes = digest.digest(userIdentifier.getBytes(StandardCharsets.UTF_8));
+    // Change this to a secret not shared with Google.
+    String HMAC_KEY = "SOME_INTERNAL_UNSHARED_KEY";
+
+    // Get instance of Mac object implementing HmacSHA256, and initialize it with the above
+    // secret key.
+    Mac mac = Mac.getInstance("HmacSHA256");
+    SecretKeySpec secretKeySpec = new SecretKeySpec(HMAC_KEY.getBytes(StandardCharsets.UTF_8),
+        "HmacSHA256");
```

Comment:
Got to wonder if you shouldn't define a constant of this, probably not, but, I wonder.

Reply:
Thanks for the catch. Better to inline with the mac.init so as to not risk exposing during debugging or logging.